mujoco_warp/_src/smooth.py

```diff
@@ -673,8 +673,14 @@ def _transmission(
         moment[worldid, actid, vadr] = gear[0]
       else:
         wp.printf("unrecognized joint type")
+    elif trntype == wp.static(TrnType.TENDON.value):
+      trnid = m.actuator_trnid[actid, 0]
+      gear0 = gear[0]
+      length[worldid, actid] = d.ten_length[worldid, trnid] * gear0
+      for i in range(m.nv):
```

Comment:
Hmmm - could you loop over `wrap_jnt_adr` instead?

Reply:
done

Reply:
fixed the fixed tendon case